data/json/recipes/recipe_food.json

```diff
@@ -10334,7 +10334,7 @@
     "tools": [ [ [ "surface_heat", 6, "LIST" ] ], [ [ "frying_oil", 1, "LIST" ] ] ],
     "//": "2u for flour, 3u for veggies, 1u for oil",
     "components": [
-      [ [ "veggy_pickled", 1, " LIST" ] ],
+      [ [ "veggy_pickled", 1, "LIST" ] ],
```

Comment:
```suggestion
      [ [ "veggy_pickled", 1 ] ],
```

Reply:
Should remove it as no such List exists in cooking_components.json (the game just ignored the " LIST" and treated it as the item veggy_pickled).